src/App.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
+import Navbar from "./Navbar";
 import logo from './sandbox_square_salmon.png';
 import './App.css';
 
 function App() {
   return (
-    <div className="App bg-red-500">
-	<button className="bg-teal-400">Button Name</button>
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Sandbox A-Eye Project Landing Page
-        </p>
-      </header>
-    </div>
+      <div className="App">
+          <div className="App-header">
+              <Navbar />
+          </div>
+
+          <img src={logo} className="App-logo" alt="logo" />
```

Comment:
Note that this is no longer centered - it'll be removed but just wanted to draw your attention to it